hack/update-deps.sh

```diff
@@ -54,11 +55,8 @@ fi
 go mod tidy
 go mod vendor
 
-rm -rf $(find vendor/ -name 'OWNERS')
-rm -rf $(find vendor/ -name 'OWNERS_ALIASES')
-rm -rf $(find vendor/ -name 'BUILD')
-rm -rf $(find vendor/ -name 'BUILD.bazel')
-rm -rf $(find vendor/ -name '*_test.go')
+find vendor/ -name 'OWNERS' -delete
+find vendor/ -name '*_test.go'-delete
 
 export GOFLAGS=-mod=vendor
 
```

Comment:
The proper way now is to use `go_update_deps`: https://github.com/knative/eventing/blob/master/hack/update-deps.sh